doc/api/tty.md

```diff
@@ -121,6 +121,29 @@ added: v0.7.7
 A `number` specifying the number of rows the TTY currently has. This property
 is updated whenever the `'resize'` event is emitted.
 
+### writeStream.getColorDepth([env])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `env` {object} A object containing the environment variables to check.
+  Defaults to `process.env`.
+* Returns: {number}
+
+Returns:
+* 1 for 2,
+* 4 for 16,
+* 8 for 256,
+* 24 for 16,777,216
+colors supported.
+
+Use this to determine what colors the terminal supports.
+Due to the nature of colors in terminals it is possible to either have false
+positives or false negatives.
+
+Use the `NODE_DISABLE_COLORS` environment variable to enforce this function to
+always return 0.
```

Comment:
Is this still true? The test seems to imply that it would return `1` if colors are disabled.

Reply:
Solved.